vcs/search_test.go

```diff
@@ -36,8 +39,18 @@ func TestRepository_Search_LongLine(t *testing.T) {
 		},
 	}
 
+	// alexsaveliev "echo .... > f1" does not work on Windows, let's create test file different way
+	tmp, err := ioutil.TempFile("", "test")
```

Comment:
Can you factor this out into a utility function, called something like `createFileWithContents(contents string)`?